### PR TITLE
New post:  Easily deploying test environments from CI builds with Travis, AWS and Slack

### PR DESCRIPTION
**Pull request by tatut**

### CODE REVIEW

File: _posts/2017-04-12-easy-test-deployments.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    ec2 = boto3.client('ec2',region_name='eu-central-1')
+    res = ec2.run_instances(ImageId='...ami-id...',
+                            InstanceType='t2.medium',
+                            UserData=script,
+                            KeyName='...keypair...',
+                            MinCount=1,
+                            MaxCount=1)
+    id = res['Instances'][0]['InstanceId']
+    host = None
+    while
```
Comment (lokori) [inline _posts/2017-04-12-easy-test-deployments.md]: Centos - CentOS

File: _posts/2017-04-12-easy-test-deployments.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
+---
+layout: post
+title: Easily deploying test environments from CI builds with Travis, AWS and Slack
+author: tatut
+excerpt: Easy setup of test environments is important for feature branch based development. This post shows how to automatically deploy a new test environment with a simple chat interface.
```
Comment (lokori) [inline _posts/2017-04-12-easy-test-deployments.md]: en tajunnut tästä excerptistä ollenkaan mistä on kyse tuossa chat interfacessa :) Kyllä se sieltä selvisi lopulta, mutta ehkä tämän voisi ilmaista jotenkin selkeämmin, jotain tällaista?
 
"This post shows how to automatically deploy builds to AWS with Travis CI and how developers can control the deployments with a a simple chat interface."

**Comment by Rinorragi** [comment]
I quick read it through. It looked interesting and I will come back to it. Although the text was a bit tense because there were not enough pictures. Moar pictures! For example you could add use case workflow picture about the deployment process. slack --> profit

File: _posts/2017-04-12-easy-test-deployments.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+
+def deploy(branch):
+    script = initscript.replace('$BRANCH',branch)
+    ec2 = boto3.client('ec2',region_name='eu-central-1')
+    res = ec2.run_instances(ImageId='...ami-id...',
+                            InstanceType='t2.medium',
+                            UserData=script,
+                            KeyName='...keypair...',
+                            MinCount=1,
+
```
Comment (Kauko) [inline _posts/2017-04-12-easy-test-deployments.md]: Safely indisctiminately, turhaan kaksi sanaa

**Pull request merged**